docs/authorization_userguide.md

```diff
@@ -0,0 +1,58 @@
+# Authorization and authentication through TiDB server
+
+Authentication refers to the class of policies and mechanisms that verify that clients are allowed to connect as a
```

Comment:
Did you mean `what clients`?

Reply:
verify that `sth need to verify`
verify that `clients are allowed to connect as a
certain user`

Reply:
Understood.